src/onnx/parse_slice.cpp

```diff
@@ -20,6 +20,8 @@ struct parse_slice : op_parser<parse_slice>
     {
         op::slice op;
 
+        bool reverse_direction = false;
```

Comment:
I think we do not even need the steps here since we did not change the slice implementation at all. We only change the way parsing the slice operator

Reply:
BTW, since we have the step operator now, we might be able to handle steps of not 1

Reply:
But we need the steps variable so we can insert step/reverse operators. It should be a variable here since we can't store it in the operators(whereas axes/start/ends are store in the slice op). We also won't store the steps directly in the step operator either since it we need to process it for reverse.

> BTW, since we have the step operator now, we might be able to handle steps of not 1

Yea, I just assume for now he was just adding reversing, but it should be done in a way we can easily insert step later on.

Reply:
****

> But we need the steps variable so we can insert step/reverse operators. It should be a variable here since we can't store it in the operators(whereas axes/start/ends are stored in the slice op). We also won't store the steps directly in the step operator either since we need to process it for reverse.
> 
> > BTW, since we have the step operator now, we might be able to handle steps of not 1
> 
> Yea, I just assume for now he was just adding reversing, but it should be done in a way we can easily insert step later on.

I had thought this is the slice operator implementation file, so I think we should not have the steps. Yes, I agree we should have the "step" in the parse_slice file.

Reply:
Makes sense, will do

Reply:
Done, added steps as well.

Reply:
I had to remove adding step operator itself. Variable is still there.